playbooks/network-ee-integration-tests/pre.yaml

```diff
@@ -3,6 +3,12 @@
   vars:
     _hosts_dict: {}
   tasks:
+    - name: Enable FIPS mode
+      include_role:
+        name: enable-fips
+      when:
+        - test_fips_mode | bool
+
```

Comment:
We need to add these conditions
- ansible_distribution == "CentOS"
 - ansible_test_network_cli_ssh_type == "libssh"

Reply:
why? can you be more specific